includes/Core/Assets/Assets.php

```diff
@@ -515,51 +515,51 @@ private function get_assets() {
 			new Script(
 				'googlesitekit-widgets',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-widgets.js',
+					'version'      => null,
 					'dependencies' => array(
 						'googlesitekit-data',
 					),
 				)
 			),
 			// End JSR Assets.
 			new Script(
-				'googlesitekit-ads-detect',
+				'pagead2.ads',
```

Comment:
Let's keep the `googlesitekit-` prefix here, the handle should still be scoped to Site Kit. What we can do instead to make it work is:
* Either add a special case for this one script hard-coded to the `Script` class.
* Or, if the handle does not exist in the manifest, strip the `googlesitekit-` prefix and try lookup again.

Reply:
Updated.